src/mixin/java/io/github/brassmc/brassloader/mixin/ClientBrandRetrieverMixin.java

```diff
@@ -0,0 +1,20 @@
+package io.github.brassmc.brassloader.mixin;
+
+import net.minecraft.client.ClientBrandRetriever;
+import net.minecraft.obfuscate.DontObfuscate;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+
+@Mixin(ClientBrandRetriever.class)
+public class ClientBrandRetrieverMixin {
+
+    /**
+     * @author BrassMC
+     * @reason change client mod name to brass
+     */
+    @DontObfuscate
```

Comment:
This annountation is not needed, it is a compile only annotation for proguard to ignore obfuscation when obfuscating Minecraft itself. 
```suggestion
```

Reply:
I added it because Intellij suggested it, but I guess its ok without